src/webviews/apps/plus/graph/GraphWrapper.tsx

```diff
@@ -464,13 +465,28 @@ export function GraphWrapper({
 
 	const handleOnColumnResized = (columnName: GraphColumnName, columnSettings: GraphColumnSetting) => {
 		if (columnSettings.width) {
-			onColumnChange?.(columnName, {
-				width: columnSettings.width,
-				isHidden: columnSettings.isHidden,
+			onColumnsChange?.({
+				[columnName]: {
+					width: columnSettings.width,
+					isHidden: columnSettings.isHidden,
+					order: columnSettings.order
+				}
 			});
 		}
 	};
 
+	const handleOnGraphColumnsReOrdered = (columnsSettings: GraphColumnsSettings) => {
+		const graphColumnsConfig: GraphColumnsConfig = {};
+		Object.keys(columnsSettings).forEach((columnName: string) => {
+			graphColumnsConfig[columnName] = {
+				width: columnsSettings[columnName].width,
+				isHidden: columnsSettings[columnName].isHidden,
+				order: columnsSettings[columnName].order
+			};
+		});
```

Comment:
`GraphColumnsSettings` and `GraphColumnsConfig` appear to be the same structure. Is the loop just to copy the objects? If so , this could be simplified to:

```suggestion
		for (const [columnName, config] of Object.entries(columnsSettings as GraphColumnsConfig)) {
			graphColumnsConfig[columnName] = { ...config };
		});
```